nomenclature/cli.py

```diff
@@ -173,3 +175,63 @@ def cli_export_definitions_to_excel(
         Path and file name for the exported file
     """
     DataStructureDefinition(path / "definitions").to_excel(target)
+
+
+@cli.command("run-workflow")
+@click.argument("input_file", type=click.Path(exists=True, path_type=Path))
+@click.option(
+    "--workflow-file",
+    default=lambda: Path.cwd() / "workflow.py",
+    type=click.Path(exists=True, path_type=Path),
+)
+@click.option("--workflow-function", default="main")
+@click.option("--output-file", type=click.Path())
+def cli_run_workflow(
+    input_file: Path,
+    workflow_file: Path,
+    workflow_function: str,
+    output_file: Path | None,
+):
+    """Run a given input file through a workflow function defined in a workflow.py
+
+    Parameters
+    ----------
+    input_file : Path
+        Input data file, must be IAMC format, .xlsx or .csv
+    workflow_file : Path
+            Path to the workflow file,
+            default: current working directory / "workflow.py"
```

Comment:
Is it on purpose that "optional" is not added to the type?
```suggestion
        Path to the workflow file,
        default: current working directory / "workflow.py"
```

Reply:
Ah no, good point, I"ll fix that.